Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -59,14 +59,16 @@ public MainWindow()
 
         private void OnCopy(object sender, ExecutedRoutedEventArgs e)
         {
-            if (QueryTextBox.SelectionLength == 0)
+            var result = _viewModel.Results.SelectedItem?.Result;
+            if (QueryTextBox.SelectionLength == 0 && result != null)
             {
-                _viewModel.ResultCopy(string.Empty);
+                string copyText = result.CopyText;
+                _viewModel.ResultCopy(copyText);
 
             }
             else if (!string.IsNullOrEmpty(QueryTextBox.Text))
             {
-                _viewModel.ResultCopy(QueryTextBox.SelectedText);
+                System.Windows.Clipboard.SetText(QueryTextBox.SelectedText);
```

Comment:
Why are we using this Clipboard method and not the viewmodel one?

Reply:
This way avoids wasting time parsing if the text is a file and doesn't send a notification. 

I don't think anyone just copying text from the query wants or needs this.

Reply:
>Copy selected text raises a notification, which is quite unnecessary. Maybe the right way is to remove notification for copy texts?

Yes agree no need for notification. I missed the method being ResultCopy where this is copying query test. 

>And BTW does this PR affects clipboard manager plugins like ClipboardR?

I haven't used the plugin, does it just access copied data or does it also monitor ctrl+c?

Reply:
I have created an issue in the ClipboardR repo to let the author know about the modification.

Reply:
>And BTW does this PR affects clipboard manager plugins like ClipboardR?

I created an issue to let the author know about this change.